files/image_config/topology/topology.sh

```diff
@@ -22,12 +22,99 @@ get_hwsku() {
     echo "${HWSKU}"
 }
 
+bind_ports_to_ns() {
+    PLATFORM="$1"
+    HWSKU="$2"
+    BIND="$3"
+
+    # Check if the directory exists
+    if [[ ! -d "/usr/share/sonic/device/$PLATFORM/$HWSKU" ]]; then
+        echo "Directory /usr/share/sonic/device/$PLATFORM/$HWSKU does not exist"
+        exit 1
+    fi
+
+    # Read NUM_ASIC from asic.conf file
+    asic_conf="/usr/share/sonic/device/$PLATFORM/asic.conf"
```

Comment:
we cannot assume all the LCs in T2 chassis is multi-asics. We need to handle single asic LC as well. For single asic LC asic.conf will not be present.

Reply:
this script gets executed from topology.service.. which is only instantiated on multi-asic devices..

Reply:
For single asic voq linecard we do not have to create the netdev for Cpu0?